taichi/codegen/llvm/codegen_llvm.cpp

```diff
@@ -2749,6 +2749,13 @@ LLVMCompiledTask TaskCodeGenLLVM::run_compilation() {
   emit_to_module();
   eliminate_unused_functions();
 
+  // CUDA specific metadata
+  for (const auto &task : offloaded_tasks) {
```

Comment:
This file is `codegen_llvm.cpp`, so this code block also marks CPU offloaded_tasks as cuda kernels?

Reply:
oops I forgot the guard, thanks for catching!